navbar-service.js

```diff
@@ -28,6 +28,9 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.brandOnly = false;
```

Comment:
So maybe `service.display = x` where 'x' can be `all`, `brand`, `menu`, `toolbar` or an array containing more than one option?